packages/in_app_purchase/in_app_purchase_storekit/example/shared/RunnerTests/FIAPPaymentQueueDeleteTests.m

```diff
@@ -0,0 +1,125 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#import <OCMock/OCMock.h>
+#import <XCTest/XCTest.h>
+#import "FIAObjectTranslator.h"
+#import "FIAPaymentQueueHandler.h"
+#import "Stubs.h"
+
+@import in_app_purchase_storekit;
+
+API_AVAILABLE(ios(13.0))
+API_UNAVAILABLE(tvos, macos, watchos)
+@interface FIAPPaymentQueueDelegateTests : XCTestCase
+
+@property(strong, nonatomic) FlutterMethodChannel *channel;
+@property(strong, nonatomic) SKPaymentTransaction *transaction;
+@property(strong, nonatomic) SKStorefront *storefront;
+
+@end
+
+@implementation FIAPPaymentQueueDelegateTests
+
+- (void)setUp {
+  self.channel = OCMClassMock(FlutterMethodChannel.class);
+
+  NSDictionary *transactionMap = @{
+    @"transactionIdentifier" : [NSNull null],
+    @"transactionState" : @(SKPaymentTransactionStatePurchasing),
+    @"payment" : [NSNull null],
+    @"error" : [FIAObjectTranslator getMapFromNSError:[NSError errorWithDomain:@"test_stub"
+                                                                          code:123
+                                                                      userInfo:@{}]],
+    @"transactionTimeStamp" : @([NSDate date].timeIntervalSince1970),
+    @"originalTransaction" : [NSNull null],
+  };
+  self.transaction = [[SKPaymentTransactionStub alloc] initWithMap:transactionMap];
+
+  NSDictionary *storefrontMap = @{
+    @"countryCode" : @"USA",
```

Comment:
Are we defaulting everyone to using the USA as their country code here?

Reply:
Just to be sure, [this comment](https://github.com/flutter/plugins/pull/6517#discussion_r1029704543) also applies here, right?
> This is just moved code; the scope of the review should be to enabling macOS support within the existing code, rather than changing/improving the code more generally.